client-course-schedulizer/src/components/reuseables/Calendar/Calendar.tsx

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable sort-imports */
+import FullCalendar, { EventInput } from "@fullcalendar/react";
+import moment from "moment";
+
+// Plugins
+import interactionPlugin from "@fullcalendar/interaction"; // needed for dayClick
+import timeGridPlugin from "@fullcalendar/timegrid";
+
+import React from "react";
+import "./Calendar.scss";
+
+export const initialDate = "2000-01-02";
```

Comment:
It seems kind of weird to need to set a date when we're not dealing with specific dates. Is there a way to make these events just with a time and not a date? It's fine if there's not.

Reply:
Unfortunately no. We can just provide functions that hide this nuance when dealing with events so we don't have to worry about it going forward.